### ISSUE TITLE
Translate to Korean

### ISSUE THREAD

**Issue by iluwatar** [iluwatar/java-design-patterns#2282]
This task tracks the translation of the patterns. More information about the translation process is available at https://github.com/iluwatar/java-design-patterns/wiki/15.-Support-for-multiple-languages

Acceptance criteria

- The patterns have been translated
- The patterns can be browsed in Github and on the website https://java-design-patterns.com

**Comment by Nickolodeon98** [comment]
Hi, could I translate `callback` pattern in Korean by any chance?

**Comment by HongGeonUi** [comment]
Hi, Im interested in this issue, can I translate some patterns in Korean?

**Comment by iluwatar** [comment]
Yes, just indicate which patterns you are going to translate, so we can avoid double work

**Comment by HongGeonUi** [comment]
> Yes, just indicate which patterns you are going to translate, so we can avoid double work

Thx, I will do this steadily. So I'll indicate when I do this work.
Now, I make the PR for Visitor.

**Comment by proceane** [comment]
Hi, I'd like to work in this issue.

**Comment by SARAELGYAR** [comment]
I'd love to work on this issue. Can you please assign it to me?

**Comment by SARAELGYAR** [comment]
I'd love to work on this issue. Can you please assign it to me?

**Comment by PlumHeadd** [comment]
Hi, I'd like to work in this issue, please assign this issue to me. thanks

**Comment by theeunseojo** [comment]
Hi, it seems there hasn't been any update on this for a while. If the current assignee is busy,  Could I contribute on this issue?

**Comment by snackcookie** [comment]
Hi, I'd be happy to help translate. Is there anything I can work on?
